worlds/mlss/Options.py

```diff
@@ -282,7 +282,7 @@ class MLSSOptions(PerGameCommonOptions):
     extra_pipes: ExtraPipes
     skip_minecart: SkipMinecart
     disable_surf: DisableSurf
-    harhalls_pants: HarhallsPants
+    disable_harhalls_pants: HarhallsPants
```

Comment:
Since you are changing the name, you may want to mark the old option name as `Removed` so that yamls with it will throw an error in generation instead of ignoring it and using the default for the new name